src/ng-select/lib/ng-select.component.ts

```diff
@@ -1,1105 +1,1134 @@
+                map(() => this._pressedKeys.join(''))
+            )
+            .subscribe((term) => {
+                const item = this.itemsList.findByLabel(term);
+                if (item) {
+                    if (this.isOpen) {
+                        this.itemsList.markItem(item);
+                        this._scrollToMarked();
+                        this._cd.markForCheck();
+                    } else {
+                        this.select(item);
+                    }
+                }
+                this._pressedKeys = [];
+            });
+    }
+
+    private _setInputAttributes() {
+        const input = this.searchInput.nativeElement;
+        const attributes = {
+            type: 'text',
+            autocorrect: 'off',
+            autocapitalize: 'off',
+            ...this.inputAttrs,
+        };
+
+        for (const key of Object.keys(attributes)) {
+            input.setAttribute(key, attributes[key]);
+        }
+    }
+
+    private _updateNgModel() {
+        const model = [];
+        for (const item of this.selectedItems) {
+            if (this.bindValue) {
+                let value = null;
+                if (item.children) {
+                    const groupKey = this.groupValue ? this.bindValue : <string>this.groupBy;
+                    value = item.value[groupKey || <string>this.groupBy];
+                } else {
+                    value = this.itemsList.resolveNested(item.value, this.bindValue);
+                }
+                model.push(value);
+            } else {
+                model.push(item.value);
+            }
+        }
+
+        const selected = this.selectedItems.map((x) => x.value);
+        if (this.multiple) {
+            this._onChange(model);
+            this.changeEvent.emit(selected);
+        } else {
+            this._onChange(isDefined(model[0]) ? model[0] : null);
+            this.changeEvent.emit(selected[0]);
+        }
+
+        this._cd.markForCheck();
+    }
+
+    private _clearSearch() {
+        if (!this.searchTerm) {
+            return;
+        }
+
+        this._changeSearch(null);
+        this.itemsList.resetFilteredItems();
+    }
+
+    private _setInputValue() {
+        if (!this.multiple && this.selectedItems.length) {
+            this.searchTerm = this.selectedItems[0].label;
+        }
+    }
+
+    private _changeSearch(searchTerm: string) {
+        this.searchTerm = searchTerm;
+        if (this._isTypeahead) {
+            this.typeahead.next(searchTerm);
+        }
+    }
+
+    private _scrollToMarked() {
+        if (!this.isOpen || !this.dropdownPanel) {
+            return;
+        }
+        this.dropdownPanel.scrollTo(this.itemsList.markedItem);
+    }
+
+    private _scrollToTag() {
+        if (!this.isOpen || !this.dropdownPanel) {
+            return;
+        }
+        this.dropdownPanel.scrollToTag();
+    }
+
+    private _onSelectionChanged() {
+        if (this.isOpen && this.deselectOnClick && this.appendTo) {

```

Comment:
Same question here as the last review comment - does replacing `multiple` here break expected behavior in MSDs?

Reply:
@geebru I will test this.

Reply:
@geebru I don't see any breaking changes within the ng-select demo, I will test again with CAweb after the release candidate release